os/fs/driver/mtd/smart.c

```diff
@@ -605,7 +605,7 @@ FAR static void *smart_malloc(FAR struct smart_struct_s *dev, size_t bytes, cons
 /****************************************************************************
  * Name: smart_free
  *
- * Description:  Perform smart memory free operation.
+ * Description:  Perform jSmartFs memory free operation.
```

Comment:
Let us not change smart to jSmartFs
We will only change smartfs to jSmartFs for now.
@Taejun-Kwon @davidfather your suggestions please?